Makefile

```
@@ -1,7 +1,7 @@
 all: start
 
 start:
-	cp src/config/env.homolog.js src/config/env.js
+	cp src/config/env.default.js src/config/env.js
```

Dentro deste arquivo tá como homolog muda para dev

```suggestion
	cp src/config/env.dev.js src/config/env.js
```